src/main/java/filter/expression/Qualifier.java

```diff
@@ -136,23 +131,61 @@ public static boolean process(IModel model, FilterExpression expr, TurboIssue is
         return exprWithNormalQualifiers.isSatisfiedBy(model, issue, new MetaQualifierInfo(metaQualifiers));
     }
 
+    /**
+     * Get all milestones that can be aliased with the keyword "current".
+     */
+    private static List<TurboMilestone> getMilestonesWithAlias(IModel model, List<String> repoIds) {
```

Comment:
`getMilestonesWithinAliasRange()`? We would like to get all milestones that can be aliased with `current-[n]` to `current+[n]`.